ooni/utils/net.py

```diff
@@ -27,7 +27,6 @@ def buildProtocol(self, addr):
 # agents with largest anonymity set.
 
 userAgents = ("Mozilla/5.0 (Windows; U; Windows NT 5.1; en-US; rv:1.9.1.7) Gecko/20091221 Firefox/3.5.7",
-              "Mozilla/5.0 (iPhone; U; CPU iPhone OS 3 1 2 like Mac OS X; en-us)"
```

Comment:
During testing we found that the user-agent was randomly selected on a per-request basis.  It was possible for the Tor request and the non-tor request to be sent with different user-agents, and if the target site was using user-agent sniffing to show a mobile-optimized site (or to redirect to a separate mobile site), the http_request test would register a false-positive for censorship.  It seemed easier to remove the iPhone user-agent to remove the ambiguity!